src/main/kotlin/lotto/model/game/Lottos.kt

```diff
@@ -1,49 +1,49 @@
-package lotto.model
+package lotto.model.game
 
+import lotto.model.result.Coincidence
 import java.math.BigDecimal
 import java.math.RoundingMode
 
-class Lottos(private val lottos: List<Lotto>) {
-    var myLottos: List<Lotto>
-        private set
-
-    init {
-        myLottos = lottos
-    }
-
+class Lottos(val lottos: List<Lotto>) {
     constructor(lottoNumPool: LottoNumberPool, count: Int) : this(createLottos(lottoNumPool, count))
 
     fun check(winningLotto: Lotto, checkCount: Int): Int {
-        return myLottos
-            .map { it.getWinningCount(winningLotto) }
+        return lottos
+            .map { it.getMatchCount(winningLotto) }
             .filter { it == checkCount }
             .count()
     }
 
     fun getEarningRate(winningLotto: Lotto): BigDecimal {
         val totalPrizeMoney = getTotalPrizeMoney(winningLotto)
-        val budgetMoney = (myLottos.size * COST_PER_ONE_LOTTO).toBigDecimal()
+        val budgetMoney = (lottos.size * COST_PER_ONE_LOTTO).toBigDecimal()
 
         return totalPrizeMoney.divide(budgetMoney, TWO_DECIMAL_PLACES, RoundingMode.FLOOR)
     }
 
+    fun getCoincidenceCount(coincidence: Coincidence, winningLotto: Lotto, bonusNumber: LottoNumber): Int {
```

Comment:
이왕 만든 `WinningLotto`를 사용하면 어떨까요?

```suggestion
    fun getCoincidenceCount(coincidence: Coincidence, winningLotto: WinningLotto): Int {
```